ql/src/test/org/apache/hadoop/hive/ql/exec/tez/TestTezTask.java

```diff
@@ -196,7 +196,7 @@ public void tearDown() throws Exception {
   }
 
   @Test
-  public void testBuildDag() throws IllegalArgumentException, IOException, Exception {
+  public void testBuildDag() throws IllegalArgumentException, Exception {
```

Comment:
Listing `IllegalArgumentException` is not necessary if `Exception` can be thrown.

Reply:
Nice catch, thanks, fixed.